src/com/amazon/corretto/crypto/provider/AmazonCorrettoCryptoProvider.java

```diff
@@ -84,6 +84,10 @@ private void buildServiceMap() {
                     .setSelfTest(AesCtrDrbg.SPI.SELF_TEST);
         }
 
+        addService("SecureRandom", "LibCryptoRng", "LibCryptoRng$SPI",
+                singletonMap("ThreadSafe", "true"), "LibCryptoRng")
+                .setSelfTest(LibCryptoRng.SPI.SELF_TEST);
+
```

Comment:
Can we also add an alias of `DEFAULT` to whatever our current default implementation is so that is will remain safe for people to always request the `DEFAULT` algorithm from our provider and know that they'll get something good?